crds/v1alpha1/src/main/crds/analysistemplates.argoproj

```diff
@@ -2704,7 +2704,7 @@ spec:
                           properties:
                             address:
                               type: string
-                            application:
+                            applicationConfig:
```

Comment:
This definitely doesn't seem right. I don't think that CRDs changed since last time I pulled them.